_config.yml

```diff
@@ -27,7 +27,7 @@ asciidoctor:
   - UserCLIBookName=CLI Guide
   - RulesDevBookName=Rules Development Guide
   - EclipseCrsGuideTitle=Eclipse and Red Hat CodeReady Studio Plugin Guide
-  - IntroToMTABookName=Introduction to the Migration Toolkit for Applications
+  - IntroTo{ProductShortName}BookName=Introduction to the Migration Toolkit for Applications
```

Comment:
Shouldnt it be Migration Toolkit for Runtimes?

Reply:
Done